packages/trader/src/Modules/Trading/Components/Form/TradeParams/barriers-list.tsx

```diff
@@ -31,7 +31,11 @@ const BarriersList = ({ className, header, onClickCross, show_table, ...props }:
                         <Text color='prominent' weight='bold' size='xs'>
                             {header}
                         </Text>
-                        <div className={`${className}__icon-close`} onClick={onClickCross}>
+                        <div
+                            className={`${className}__icon-close`}
+                            data-testid={`dt_${className}__icon-close`}
+                            onClick={onClickCross}
+                        >
                             <Icon icon='IcCross' />
```

Comment:
`Icon` accepts `data-testid` prop (line 56 _packages/components/src/components/icon/icon.tsx_), so you can pass it instead of adding it on `div`

Reply:
..or replace this div with button component

Reply:
Sure, can try that